fetchmail_notify_error_to_sender/__manifest__.py

```diff
@@ -0,0 +1,27 @@
+# Copyright 2015 Lorenzo Battistini <lorenzo.battistini@agilebg.com>
+# Copyright 2017 Eficent <http://www.eficent.com>
+# Copyright 2018 Hai Dinh <haidd.uit@gmail.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    'name': 'Fetchmail Notify Error to Sender',
+    'summary': 'If fetching mails gives error, send an email to sender',
+    'version': '12.0.1.0.0',
+    'category': 'Tools',
+    'author':
+        "Agile Business Group,Eficent,Odoo Community Association (OCA)",
+    'website': 'https://github.com/OCA/server-tools',
+    'license': 'AGPL-3',
+    'depends': [
+        'fetchmail',
+        'test_mail'
```

Comment:
I find it weird to require installing a module that is only meant for testing purposes. It would clutter a production database with useless models, possibly adding some weight for migrations. I'm not saying it's wrong, but do you think there could be a better approach?  :thinking: I have seen a couple of interesting clues:

- [The `hr` module uses this one for tests](https://github.com/odoo/odoo/blob/691dc424a1d04c7ea4fb87663b17922efd5bcaf5/addons/hr/tests/common.py#L4) [without adding it to its dependencies](https://github.com/odoo/odoo/blob/691dc424a1d04c7ea4fb87663b17922efd5bcaf5/addons/hr/__manifest__.py#L18-L23). Would this work?
- In the case of the `mass_mailing` addon, there's a separate addon for doing the tests, [which depends on the addon being tested (`mass_mailing`) and the addon providing the test structure (`test_mail`)](https://github.com/odoo/odoo/blob/691dc424a1d04c7ea4fb87663b17922efd5bcaf5/addons/test_mass_mailing/__manifest__.py#L12). What do you think about doing it like that?